Makefile

```diff
@@ -100,3 +100,8 @@ CHART_BRANCH ?= main
 update-dev-chart:
 	GOPROXY=direct go get -u -v github.com/percona/percona-helm-charts/charts/everest@$(CHART_BRANCH)
 	go mod tidy
+
+EVEREST_OPERATOR_BRANCH ?= main
```

Comment:
_(This comment is not related to the selected line, just wanted to have a thread somewhere)_
 
Would we need to handle the cleanup for the `everestv1alpha1.ReadOnlyFinalizer` when uninstalling Everest?

Reply:
it is handled here https://github.com/percona/percona-helm-charts/pull/531/files#diff-862e8fe6ac29a2fff8ac4afac8a98a7652d7b818e567fe27a94eb82c2f2f7387